package.json

```
@@ -49,6 +49,7 @@
     "node-filter-async": "1.1.3",
     "npm-check-updates": "3.2.1",
     "oclif": "1.14.2",
+    "parse-commit-message": "^3.2.3",
```

```suggestion
    "parse-commit-message": "3.2.3",
```
pin to the latest version